.github/ISSUE_TEMPLATE/cli_feature.yml

```diff
@@ -1,4 +1,4 @@
-name: Buildtest Command Line Feature Request
+name: Buildtest Feature Request
```

Comment:
i think we can remove `Buildtest` and just say `Feature Request`. Its already implied